--- ./zephyr_git-v1.10.0/boards/x86/galileo/pinmux.c	2017-12-08 12:32:22.000000000 -0700
+++ ./zephyr-zephyr-v1.10.0/boards/x86/galileo/pinmux.c	2018-03-08 15:52:06.083813807 -0700
@@ -61,12 +61,12 @@
 };
 
 static struct mux_path _galileo_path[PINMUX_NUM_PINS * NUM_PIN_FUNCS] = {
-	{0, PINMUX_FUNC_A, {{ EXP1,  0,  PIN_HIGH, (GPIO_DIR_OUT) }, /* GPIO3 out */
+	{0, PINMUX_FUNC_A, {{ EXP1,  0,  PIN_LOW, (GPIO_DIR_OUT) }, /* GPIO3 out */
 			    { EXP1,  1,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { G_DW,  3,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  } } },
-	{0, PINMUX_FUNC_B, {{ EXP1,  0,   PIN_LOW, (GPIO_DIR_OUT) }, /* GPIO3 in */
+	{0, PINMUX_FUNC_B, {{ EXP1,  0,   PIN_HIGH, (GPIO_DIR_OUT) }, /* GPIO3 in */
 			    { EXP1,  1,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { G_DW,  3,   PIN_LOW, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
@@ -103,12 +103,12 @@
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  } } },
 
-	{2, PINMUX_FUNC_A, {{ PWM0, 13,  PIN_HIGH, (GPIO_DIR_OUT) }, /* GPIO5 out */
+	{2, PINMUX_FUNC_A, {{ PWM0, 13,  PIN_LOW, (GPIO_DIR_OUT) }, /* GPIO5 out */
 			    { EXP1,  2,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { EXP1,  3,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { G_DW,  5,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  } } },
-	{2, PINMUX_FUNC_B, {{ PWM0, 13,  PIN_HIGH, (GPIO_DIR_OUT) }, /* GPIO5 in */
+	{2, PINMUX_FUNC_B, {{ PWM0, 13,  PIN_LOW, (GPIO_DIR_OUT) }, /* GPIO5 in */
 			    { EXP1,  2,  PIN_HIGH, (GPIO_DIR_OUT) },
 			    { EXP1,  3,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { G_DW,  5,   PIN_LOW, (GPIO_DIR_IN)  },
@@ -265,7 +265,7 @@
 			    { EXP0,  7,   PIN_LOW, (GPIO_DIR_OUT) },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  } } },
-	{9, PINMUX_FUNC_C, {{ NONE,  0, DONT_CARE, (GPIO_DIR_IN)  }, /* NONE */
+	{9, PINMUX_FUNC_D, {{ NONE,  0, DONT_CARE, (GPIO_DIR_IN)  }, /* NONE */
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
 			    { NONE,  0, DONT_CARE, (GPIO_DIR_IN)  },
@@ -592,14 +592,14 @@
 	/* Analog Inputs */
 	{ 0,  PINMUX_FUNC_C }, /* GPIO3 (out), GPIO3 (in), UART0_RXD, NA */
 	{ 1,  PINMUX_FUNC_C }, /* GPIO4 (out), GPIO4 (in), UART0_TXD, NA */
-	{ 2,  PINMUX_FUNC_C }, /* GPIO5 (out), GPIO5 (in), UART1_RXD, NA */
+	{ 2,  PINMUX_FUNC_A }, /* GPIO5 (out), GPIO5 (in), UART1_RXD, NA */
 	{ 3,  PINMUX_FUNC_C }, /* GPIO6 (out), GPIO6 (in), UART1_TXD, PWM.LED1 */
 	{ 4,  PINMUX_FUNC_B }, /* GPIO_SUS4 (out), GPIO_SUS4 (in), NA, NA */
 	{ 5,  PINMUX_FUNC_B }, /* GPIO8 (out), GPIO8 (in), PWM.LED3, NA */
 	{ 6,  PINMUX_FUNC_B }, /* GPIO9 (out), GPIO9 (in), PWM.LED5, NA */
 	{ 7,  PINMUX_FUNC_A }, /* EXP1.P0_6 (out), EXP1.P0_6 (in), NA, NA */
 	{ 8,  PINMUX_FUNC_A }, /* EXP1.P1_0 (out), EXP1.P1_0 (in), NA, NA */
-	{ 9,  PINMUX_FUNC_B }, /* GPIO_SUS2 (out), GPIO_SUS2 (in), PWM.LED7, NA */
+	{ 9,  PINMUX_FUNC_C }, /* GPIO_SUS2 (out), GPIO_SUS2 (in), PWM.LED7, NA */
 #if defined(CONFIG_SPI_1)
 	{ 10, PINMUX_FUNC_A }, /* GPIO2 (out), GPIO2 (in), PWM.LED11, NA */
 	{ 11, PINMUX_FUNC_D }, /* GPIO_SUS3 (out), GPIO_SUS3 (in), PWM.LED9, SPI1_MOSI */
